solidity/contracts/LPRewards.sol

```diff
@@ -100,7 +103,11 @@ contract LPTokenWrapper {
     }
 }
 
-contract LPRewards is LPTokenWrapper, IRewardDistributionRecipient {
+contract LPRewards is
+    LPTokenWrapper,
+    IRewardDistributionRecipient,
+    IStakerRewards
```

Comment:
It is no longer needed, `LPRewards` will receive tokens via `notifyRewardAmount` called by `StakingPoolRewardsEscrowBeneficiary` (see https://github.com/keep-network/keep-core/pull/2230) - it is the same mechanism as for `CurveRewards`.

Reply:
Does it mean that it has to implement `IStakingPoolRewards` instead of `IStakerRewards`?

Reply:
Yeah, it can implement it. I'd say it does not have to but we are also fine with implementing it ;-)